.circleci/config.yml

```diff
@@ -35,34 +35,30 @@ jobs:
       - attach_workspace:
           at: ~/build
       - run: cp ~/build/helmfile ~/project/helmfile
-      - run:
-          name: Install kubectl
-          command: |
-            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.4/bin/linux/amd64/kubectl
-            chmod +x kubectl
-            sudo mv kubectl /usr/local/bin/
-      - run:
-          name: Install minikube
-          command: |
-            curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.0/minikube-linux-amd64
-            chmod +x minikube
-            sudo mv minikube /usr/local/bin/
       - run:
           name: Install helm
+          environment:
+            HELM_VERSION: v2.9.0
           command: |
-            HELM_VERSION=v2.8.2
             HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
             curl -Lo ${HELM_FILENAME} "https://kubernetes-helm.storage.googleapis.com/${HELM_FILENAME}"
             tar zxf ${HELM_FILENAME} linux-amd64/helm
             chmod +x linux-amd64/helm
             sudo mv linux-amd64/helm /usr/local/bin/
       - run:
-          name: Start minikube
+          name: Deploy minikube
+          environment:
+            CHANGE_MINIKUBE_NONE_USER: true
+            K8S_VERSION: v1.9.0
+            MINIKUBE_VERSION: v0.25.2
           command: |
-            sudo minikube start --vm-driver=none
-            sudo chown -R $USER.$USER ~/.minikube
-            sudo chown -R $USER.$USER ~/.kube
-            minikube update-context
+            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl
+            chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+            curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
+            chmod +x minikube && sudo mv minikube /usr/local/bin/
+            sudo minikube config set WantReportErrorPrompt false
+            sudo -E minikube start --vm-driver=none --bootstrapper=localkube --kubernetes-version=${K8S_VERSION}
```

Comment:
Ah. I heard that recent minikube started to rely on kubeadm to bootstrap k8s by default. Perhaps you reverted it to the old behavior with `--bootstrapper=localkube`.

Reply:
@mumoshu Yes, it's a remnant of trying to run the tests with minikube `v0.26.1`. It looks like they switched to kubeadm in 0.26.x and require `systemd` to be installed on the host for `vm-driver=none` to work. The CircleCI image we use does not come with it, so I had to use a different bootstrapper, but I could not get it to work. So I reverted back to 0.25.x, which makes the localkube setting redundant. I removed it now, good catch.